mutt.h

```diff
@@ -26,9 +26,6 @@
 #ifdef HAVE_UNISTD_H
 # include <unistd.h> /* needed for SEEK_SET */
 #endif
-#ifdef HAVE_UNIX_H
-# include <unix.h>   /* needed for snprintf on QNX. */
```

Comment:
I'm not entirely sure, but if this allows us to keep compatibility with some esoteric platform, then it should be okay. 3 lines of code here won't hurt us much.

Does anyone know if Neomutt still compiles on QNX and if this ifdef is required on it?